docs/release/release-tasks.md

```diff
@@ -215,6 +218,7 @@ to a newer Go minor version according to our [backport policy](./../../CONTRIBUT
         * `kpromo` uses `git@github.com:...` as remote to push the branch for the PR. If you don't have `ssh` set up you can configure
           git to use `https` instead via `git config --global url."https://github.com/".insteadOf git@github.com:`.
         * This will automatically create a PR in [k8s.io](https://github.com/kubernetes/k8s.io) and assign the CAPI maintainers.
+        * Wait for the [promotion prow job](https://prow.k8s.io/?repo=kubernetes%2Fk8s.io&job=post-k8sio-image-promo) to complete successfully before going to the next step.
```

Comment:
I think this should happen after the merge. Merge is step 4

Reply:
great point. I will do that.